Sources/PushNotifications.swift

```diff
@@ -9,13 +9,13 @@ import Foundation
 
 @objc public final class PushNotifications: NSObject {
     private let session = URLSession.shared
```

Comment:
I think we might want to create a new one so that we don't use the application `URLSession.shared`...

What do you think?